src/features/lightspeed/errors.ts

```diff
@@ -117,34 +117,9 @@ export const ERRORS_CONNECTION_TIMEOUT = new Error(
   "fallback__connection_timeout",
   "Ansible Lightspeed connection timeout. Please try again later.",
 );
-
-ERRORS.addError(
-  204,
-  new Error(
-    "postprocess_error",
-    "An error occurred post-processing the inline suggestion. Please contact your administrator.",
-  ),
-);
-ERRORS.addError(
-  204,
-  new Error(
-    "model_timeout",
-    "Ansible Lightspeed timed out processing your request. Please try again later.",
-  ),
-);
-ERRORS.addError(
-  204,
-  new Error(
-    "error__wca_bad_request",
-    "IBM watsonx Code Assistant returned a bad request response. Please contact your administrator.",
-  ),
-);
-ERRORS.addError(
-  204,
-  new Error(
-    "error__wca_empty_response",
-    "IBM watsonx Code Assistant returned an empty response. Please contact your administrator.",
-  ),
+export const ERRORS_EMPTY_RESPONSE = new Error(
+  "empty__response",
+  "Ansible Lightspeed didn't find suggestion for your request. Please try reword.",
```

Comment:
Is this the message we used to show for `204`'s?

The four possible `204`'s we return represent different scenarios.

IDK if a more generic message should be used?

Reply:
Fair enough.. @robinbobbitt is usually very good at suggesting wording.

I'm not one to get into length discussions about it.. and she usually changes what I propose anyway :smile_cat: 

I assume changing the 204's in the server side is still work to be completed, or is it [dying a death](https://en.wiktionary.org/wiki/die_a_death#:~:text=To%20fail%20or%20flop%3B%20to%20lose%20popularity.) now?

Reply:
@hasys I was thinking something like "Ansible Lightspeed does not have a suggestion for this input. Try changing your prompt, or contact your administrator with Suggestion Id the_suggestion_id for assistance." wdyt?

@manstis my hope was we could revisit the 204 status code when we GA the API, i.e. v1.

Reply:
Thank you @manstis, @robinbobbitt, fixed.